### DISCUSSION TITLE
Failed With Status 405 - Method Not Allowed

### DISCUSSION THREAD

**Discussion by LuisVilaplana** [oqtane/oqtane.framework#4646] [General] [answered]
I'm getting this error on a fresh install of Oqtane 5.21 on IIS and SQL Server on Windows Server 2019. I can't delete modules from pages or even entire pages. Any idea how to fix this?

Failed With Status 405 - Method Not Allowed

Update: Adding a new page leads me  to a blank site and the only way to gain control again is by complete delete and reinstall the entire site

Thanks

**Comment by thabaum** [comment]
Hello @LuisVilaplana did you set permissions on all the folders so IIS App Pool can make these adjustments?  Oqtane docs help cover how it can be setup.  The upcoming 5.2.2 to be release has some great enhancements and fixes that should help resolve IIS issues reported earlier.

What steps are you using to set up your environment?

Are they similar to these: [Install Oqtane Using Windows, IIS and MS SQL Server ](https://docs.oqtane.org/manuals/admin/installation/windows-iis-sql.html)

Please report anything that can help enhance these docs to the Oqtane.Docs repository if you feel something can help enhance these instructions for you.

**Reply by LuisVilaplana** [reply]
Hello, yes all folders have full permissions for user IWPD_****, I have also tested setting LocalSystem in the app pool's Identity Model with no luck. By the way, site was created in Plesk panel.

When is version 5.22 coming out?

Thanks

**Reply by leigh-pointer** [reply, marked as answer]
I think you are experiencing issues with webDAV. This service should be disabled. I posted about this before with solution that can be added to your web config file.

**Reply by LuisVilaplana** [reply]
You were right Leigh, WebDav was on, it was supposed to be disabled at server level, but I was still able to disable it for that site in IIS by deleting it from Handler Mappings and Modules

Thanks a lot

**Reply by thabaum** [reply]
Hi @LuisVilaplana,  Glad that solution worked.

Thanks for your answer! @leigh-pointer, could we consider adding this to the Oqtane.Docs repository? It would be great to have these insights documented, as it can really help others in the future. Repeated questions can lead to frustration, so having a clear reference would benefit everyone, especially vendors looking to support their clients effectively.

While I personally don’t rely on the documentation, I believe it’s crucial for the community’s growth. I plan to sift through past discussions this winter to help enhance the docs, but I recognize that @iJungleboy has been doing a lot of the heavy lifting lately, and I feel we could use more collaboration.

Even posting a link to solutions in discussions could make a big difference. @LuisVilaplana, if you could share your findings with screenshots for Plesk, that would be incredibly helpful, as I don’t have access to control panels.

Thanks again for your support! Let’s keep building a strong documentation resource together.

**Reply by leigh-pointer** [reply]
> You were right Leight, WebDav was on, it was supposed to be disabled at server level, but I was still able to disable it for that site in IIS by deleting it from Handler Mappings and Modules
> 
> Thanks a lot
Very welcome 
Please mark as answered
